markdown.dtx

```diff
@@ -15994,6 +16192,137 @@ following text:
 %
 % \begin{markdown}
 
+#### Tex Math Renderers
+The \mdef{markdownRendererInlineMath} and \mdef{markdownRendererDisplayMath} macros 
+represent inline and display \TeX{} math. 
+Both macros receive a single argument that corresponds to the tex math content.
+These macros will only be produced, when the \Opt{texMathDollars}
+option is enabled.
+
+% \end{markdown}
+%
+% \iffalse
+
+##### Plain \TeX{} Example {.unnumbered}
+
+Using a text editor, create a text document named `document.tex` with the
+following content:
+``` tex
+\input markdown
+\def\markdownOptionTexMathDollars{true}
+\def\markdownRendererInlineMath#1{$\underline{#1}$}
```

Comment:
`\underline` is a plain TeX primitive that applies to a single math atom, not an entire formula.